packages/tailwind-composer/src/types.ts

```
@@ -0,0 +1,411 @@
+    input?: SlottedVariantInputValue<TSlots, TVariants> &
+      SlottedClassPropOptional<TSlots>
+  ): {
+    [K in keyof TSlots]: (
+      input?: SlottedVariantInputValue<TSlots, TVariants> & ClassPropOptional
+    ) => string;
+  };
+
+  extend: <
+    TNewSlots extends SlotMap,
+    // eslint-disable-next-line @typescript-eslint/no-empty-object-type
+    TNewVariants extends SlottedVariants<MergeSlots<TSlots, TNewSlots>> = {},
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)